services/styles.js

```diff
@@ -368,22 +380,24 @@ Styles.getFontGlyphRange = function(config) {
  * See [the corresponding HTTP service documentation](https://docs.mapbox.com/api/maps/#request-embeddable-html).
  *
  * @param {Object} config
- * @param {string} styleId
- * @param {boolean} [scrollZoom=true] - If `false`, zooming the map by scrolling will
+ * @param {string} config.styleId
+ * @param {boolean} [config.scrollZoom=true] - If `false`, zooming the map by scrolling will
  *   be disabled.
- * @param {boolean} [title=false] - If `true`, the map's title and owner is displayed
+ * @param {boolean} [config.title=false] - If `true`, the map's title and owner is displayed
  *   in the upper right corner of the map.
- * @param {ownerId} [ownerId]
+ * @param {string} [config.ownerId]
+ * @param {boolean} [config.draft=false] If `true` will retrieve the draft style, otherwise will retrieve the published style.
  */
 Styles.getEmbeddableHtml = function(config) {
   v.assertShape({
     styleId: v.required(v.string),
     scrollZoom: v.boolean,
     title: v.boolean,
-    ownerId: v.string
+    ownerId: v.string,
+    draft: v.boolean
   })(config);
 
-  var fileName = config.styleId + '.html';
+  var fileName = config.styleId + (config.draft ? '/draft' : '') + '.html';
```

Comment:
When I was testing locally, the `/` in `/draft` was getting URI encoded. Do you think this is another case like https://github.com/mapbox/mapbox-sdk-js/pull/352#issuecomment-578498228?

Reply:
oh good catch, you're right.